products/idn/api/authorization.md

```diff
@@ -0,0 +1,128 @@
+
+:::tip
+
+Because scopes can only grant access to APIs that are a part of the rights included in a user level, it is often easier to assign the **Admin** user level to the user first and then apply scopes to the OAuth credentials to limit the access.
+
+:::
+
+### Identifying Necessary Authorization for an Endpoint
+
+Each endpoint document specifies how to authorize with the endpoint in the **Authorization** dropdown, which is located on the right side column below the endpoint path. Selecting **Authorization** expands the dropdown menu showing the details of how to authorize with the endpoint. The following image shows the authorization details of the [List Access Profiles](https://developer.sailpoint.com/idn/api/beta/list-access-profiles) endpoint.
+
+![Authorization Dropdown](./img/authorization/authorization-dropdown.png)
+
+- **type**: The type of authorization method supported for this endpoint. SailPoint uses the [OAuth2 standard](./authentication.md#oauth-20) for all the V3/Beta APIs.
+- **flow**: One or more OAuth flows supported by the endpoint. A token only needs to be generated by one flow to be valid. Refer to [Authentication Details](./authentication.md#authentication-details) for more information about the available flows.
+- **scopes**: A list of scopes necessary to access the endpoint. A token only needs one of the scopes to authorize with the endpoint. When possible, choose the least privileged scope. Scopes ending in `read` can only retrieve data. Scopes ending in `manage` can retrieve, modify, and delete data.
+
+:::info
+
+SailPoint is working to define scopes for every endpoint, but you may encounter a scenario where you need access to an endpoint that does not yet have a scope defined. Until a scope is defined for the endpoint, you can assign `sp:scopes:all` to ensure that your credentials can access the necessary endpoints. Once all of the endpoints necessary for your use case have scopes defined, you can create a new credential with the appropriate scopes in place.
+
+:::
+
+### Assigning Scopes with the UI
+
+TODO: Ask Jennifer Mitchel if the UI docs are ready.
+
+### Assigning Scopes with the API
+
+You can create PATs ([personal access tokens](https://developer.sailpoint.com/idn/api/v3/create-personal-access-token)) and [OAuth clients](https://developer.sailpoint.com/idn/api/v3/create-oauth-client) programmatically with the API. The request body for each API endpoint allows the caller to specify a list of scopes to be applied to the credentials. If the `scope` property is omitted from the request body, then `sp:scopes:all` is granted to the credentials. If you try to add a scope outside the permissions of the [target user's level](#user-level-permissions), the request still succeeds, but it excludes the offending scope. The following example shows how to generate a personal access token with the `idn:access-request:manage` and `idn:nelm:manage` scopes.
+
```

Comment:
"it excludes the offending scope" doesn't feel quite correct to me. The PAT or API client will still show that scope, but tokens generated from it will not have the rights associated with the offending scope.  Because the rights are the intersection of scope with the user's role.

Maybe something like "the request still succeeds, but tokens generated from the PAT or oauth client will not be able to access APIs that require the offending scope"?

Reply:
Thanks for catching that.  I took your suggestion and moved that info to the bottom of the section inside a "caution" block.  It will read as follows:

_If you attempt to add a scope that is outside the permissions of the [target user's level](#user-level-permissions), then the request will still succeed and include the invalid scope in the credentials.  However, any token generated with these credentials will **not** include the rights of the invalid scope.  This isn't an issue when assigning scopes via the user interface, as the UI will only show scopes available to the current user._

Reply:
When will the endpoint that returns available scopes for a user be public?  I can reference that endpoint as a solution.

Reply:
It is available. /beta/authorization-scopes?requested-for=me
Looks like we don't yet have API docs for that though :-( 
I can open a ticket for that.